.wuf/generator/templates/Component/Component.e2e.ts

```diff
@@ -1,15 +1,12 @@
 import { browser } from 'protractor';
```

Comment:
I was very close to removing this file, and I agree with the points you mentioned.
It's just that adding an e2e test can be a bit complicated.
So I thought leaving this file can make it easier.

But yeah, I think I'll remove the file.
I'll add a document to explain how to add e2e tests in case one's needed.

Reply:
we'll do it in a separate PR